example_test.go

```diff
@@ -20,15 +20,29 @@ func ExampleParse(myToken string, myLookupKey func(interface{}) (interface{}, er
 
 func ExampleNew(mySigningKey []byte) (string, error) {
 	// Create the token
-	token := jwt.New(jwt.SigningMethodHS256)
+	token := jwt.New(jwt.SigningMethodRS256)
+
 	// Set some claims
-	token.Claims["foo"] = "bar"
-	token.Claims["exp"] = time.Now().Add(time.Hour * 72).Unix()
+	claims := token.Claims.(jwt.MapClaim)
+	claims["foo"] = "bar"
+	claims["exp"] = 15000
```

Comment:
What I'll do is use a fixed time rather than time.Now(), meet in the middle Solomon style :)